cif_pow.dic

```diff
@@ -11967,6 +11967,32 @@ save_pd_proc_ls.diffractogram_id
 
 save_
 
+save_pd_proc_ls.number_parameters
+
+    _definition.id                '_pd_proc_ls.number_parameters'
+    _definition.update            2025-07-20
+    _description.text
+;
+    Number of parameters refined in the least-squares process. If
```

Comment:
I believe this has been defined in the per-diffractogram `pd_proc_ls` so that the Q parameter can be calculated, in which case we need to explain what a per-diffractogram number of parameters means. If it is instead the total number of parameters over all diffractograms, then the total number of parameters will be defined elsewhere, but can still be used in the dREL equation.